wasm2c/wasm-rt-impl.c

```diff
@@ -168,6 +169,7 @@ static int os_mprotect(void* addr, size_t size) {
 static void os_print_last_error(const char* msg) {
   perror(msg);
 }
+#endif
```

Comment:
@keithw @shravanrn  WDYT about this change?

Reply:
Sure, this should be fine assuming the existing tests pass.